Makefile

```diff
@@ -0,0 +1,49 @@
+default: build
+all: build
+
+ICED=node_modules/.bin/iced
+BUILD_STAMP=build-stamp
+TEST_STAMP=test-stamp
+WD=`pwd`
+BROWSERIFY=node_modules/.bin/browserify
+
+lib/%.js: src/%.iced
+	$(ICED) -I browserify -c -o `dirname $@` $<
+
+$(BUILD_STAMP): \
+	lib/nonce.js \
+	lib/header.js
```

Comment:
Any particular reason these dependencies are split into new lines? It makes it a little tricky to tell them apart from the actual command below.